src/main/java/com/zenuml/confluence/macro/SequenceMacro.java

```diff
@@ -34,27 +34,27 @@ public SequenceMacro(
         this.settingsManager = settingsManager;
     }
 
-    public String createMD5(String plaintext) {
+    private String createMD5(String plaintext) {
         try {
             MessageDigest md = MessageDigest.getInstance("MD5");
             md.update(plaintext.getBytes());
             byte[] digest = md.digest();
             String myHash = DatatypeConverter.printHexBinary(digest).toLowerCase();
             return  myHash;
         } catch (NoSuchAlgorithmException e) {
-            return null;
+            throw new RuntimeException(e);
```

Comment:
Not the best practice to wrap with RuntimeException. But I am fine to merge this unless others have different opinions. @mwpeng 
It should be wrapped with a Business Oriented exception such as FailedToGenerateMD5Exception.